acceptance/acceptance_test.go

```diff
@@ -45,6 +46,7 @@ var (
 	LogRequests bool
 	LogConfig   bool
 	SkipLocal   bool
+	Dbr         bool
```

Comment:
We could implicitly also rely on the DATABRICKS_RUNTIME_VERSION environment variable being set. But this feels more robust.

Either approach is good enough, though IMO.